DjangoProject/bookmarksapp/models.py

```diff
@@ -1,3 +1,30 @@
 from django.db import models
+from django.utils.text import slugify
+
+from model_utils.models import TimeStampedModel
+
+from authenticationapp.models import MyUser
 
 # Create your models here.
+
+
+class Folders(TimeStampedModel):
+    name = models.CharField(max_length=50, unique=True)
+    slug = models.SlugField(unique=True)
+    user = models.ForeignKey(MyUser, on_delete=models.CASCADE)
+
+    def save(self, *args, **kwargs):
+        self.slug = slugify(self.name)
+        super(Folders, self).save(*args, **kwargs)
+
+
+class Bookmarks(TimeStampedModel):
+    folder = models.ForeignKey(Folders, on_delete=models.SET_NULL, related_name='folder', null=True)
+    name = models.CharField(max_length=50, unique=True)
+    url = models.URLField(max_length=200, unique=True)
+    user = models.ForeignKey(MyUser, on_delete=models.CASCADE, default=1)
```

Comment:
Why do u have a user field here? Field name should be verbose. If it's for storing the user who created the bookmark it should be `created_by`. Why is `default=1`?